UniAtHome/UniAtHome.DAL/UniAtHomeDbContext.cs

```diff
@@ -1,9 +1,75 @@
-﻿using Microsoft.EntityFrameworkCore;
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using UniAtHome.DAL.Entities;
 
 namespace UniAtHome.DAL
 {
     public class UniAtHomeDbContext : DbContext
     {
-        public UniAtHomeDbContext(){}
+        public UniAtHomeDbContext(DbContextOptions<UniAtHomeDbContext> options) : base(options)
+        {
+        }
+
+        public DbSet<Course> Courses { get; set; }
+        public DbSet<Lesson> Lessons { get; set; }
+        public DbSet<User> Users { get; set; }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            base.OnModelCreating(modelBuilder);
+            modelBuilder.Seed();
+
+            modelBuilder.Entity<Course>().HasKey(c => c.Id);
```

Comment:
I'm going to use only FluentAPI without annotation attributes. What do you think?

Reply:
Entity framework's annotations succ  😎